test/compilable/diag13110b.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+name=`basename $0 .sh`
+dir=${RESULTS_DIR}/compilable
+output_file=${dir}/${name}.sh.out
+
+function echorun() {
+    echo $@ > ${output_file}
+    $@
+}
+
+out=$(echorun ${DMD} -conf= -m${MODEL} compilable/extra-files/echoargs.d -run a b c)
+[ "$out" == "a b c" ] || exit 1
```

Comment:
`... | grep -q "a b c"` would work too.

Reply:
looks like `grep` works better since windows doesn't cut of the extra `\r`